test/unit/css.js

```diff
@@ -1753,9 +1729,6 @@ QUnit.test( "Do not throw on frame elements from css method (#15098)", function(
 
 		assert.expect( !!appearanceName + !!transformName + 1 );
 
-		resetCssPropsFor( "appearance" );
-		resetCssPropsFor( "transform" );
```

Comment:
This does make the test a little less isolated but it's already been that way since `finalPropName` has been refactored by @dmethvin to use an internal `vendorProps` map so we're not losing much here.

Another test can influence this one if it sets some transform- or appearance-related props. We can think if there's any way around that but, as I said, it's already an issue.